README.md

```diff
@@ -16,13 +16,26 @@ Installation does not assume any previous packages having been installed. Thus t
 
 `"eslint-config-vocovo": "https://github.com/vocovo/eslint-config-vocovo",`
 
+and run `yarn` to install it
+
 3. Add these lines to your npm `scripts` section:
 
 ```
 "lint": "eslint --ext .js src",
 "lint:ci": "eslint -f summary --ext .js src",
 ```
 
+4. Add the following `.eslintrc` to your project root
+
+```
+{
+  "extends": ["vocovo"],
+  "env": {
```

Comment:
Any env variables here are only necessary if you're using those features, they're not a requirement of this package. And similarly adding `"parser": "babel-eslint",` might be necessary, but only if you're using features that require transpilation to understand. 

It's hard to know how much/little to add to a readme here. Can't disagree with adding `"extends": ["vocovo"],` though, that's defo essential